jwst/tweakreg/tweakreg_step.py

```diff
@@ -311,12 +323,12 @@ def process(self, input):
                 # from overlapping images where centering is not consistent or
                 # for the possibility that errors still exist in relative overlap.
                 tpmatch_gaia = TPMatch(
-                    searchrad=self.searchrad * 3.0,
-                    separation=self.separation / 10.0,
-                    use2dhist=self.use2dhist,
-                    tolerance=self.tolerance,
-                    xoffset=0.0,
-                    yoffset=0.0
+                    searchrad=self.abs_searchrad,
+                    separation=self.abs_separation,
+                    use2dhist=self.abs_use2dhist,
+                    tolerance=self.abs_tolerance,
+                    xoffset=self.xoffset if self.abs_use2dhist else 0.0,
+                    yoffset=self.yoffset if self.abs_use2dhist else 0.0
```

Comment:
I think we need separate `abs_xoffset` and `abs_yoffset` since we have a separate (for "absolute astrometry" step) `abs_use2dhist` parameter.

Reply:
Do you have a guess for what values should we use for these offsets?

Reply:
Not sure I understood the question, but it should default to 0 if not provided, just like parameters for the "relative alignment":
```python
        xoffset = float(default=0.0), # Initial guess for X offset in arcsec
        yoffset = float(default=0.0) # Initial guess for Y offset in arcsec
```

Reply:
`abs_xoffset` and `abs_yoffset` are not supposed to be exposed, which means their value must be hardcoded. So the question is do we have any good estimate for that?

Reply:
If we hardcode it to zero and use `abs_xoffset` instead of `xoffset`, then we won't need this: `xoffset=self.abs_xoffset if self.abs_use2dhist else 0.0` in the call to `TPMatch`. In the ticket @antonkoekemoer says the following:

> If abs_use2dhist is turned off, then just set xoffset,yoffset = 0,0, in the code (and then, in the parameter description, you can add text to strongly encourage users to set abs_use2dhist=True, and inform them that if it's False then the initial guess for the shifts will simply be set to 0,0)

That's why I was doing `xoffset=self.xoffset if self.abs_use2dhist else 0.0`.

Should we use `self.xoffset` whenever `self.abs_use2dhist` is set to `False` or should we hardcode `self.abs_xoffset=0` in the code and update the call to `TPMatch` to use `xoffset=self.abs_xoffset`?

Reply:
Set `self.abs_xoffset` and `self.abs_yoffset` to zero.